client/post-editor/media-modal/detail/detail-item.jsx

```diff
@@ -200,7 +200,12 @@ export class EditorMediaModalDetailItem extends Component {
 		}
 
 		return (
-			<EditorMediaModalDetailFields site={ site } item={ item } onUpdate={ this.props.onUpdate } />
+			<EditorMediaModalDetailFields
+				key={ `detail-fields-${ item?.ID }` }
```

Comment:
Should we also include the side ID in the `key`? That way we'd properly get a new component instance if the media ID is the same across two different sites.

```suggestion
				key={ `detail-fields-${ this.props.siteId }-${ item?.ID }` }
```

Reply:
Added via https://github.com/Automattic/wp-calypso/pull/62568/commits/685a6f997a305ceb59218c12dbb9048559381b19 which uses the desctructured prop. I'm not exactly sure it's  necessary because I don't think it's possible for users to change the currently active site without also closing the modal but it's no harm to be specific in this case 👍🏻